discovery_engine/test/feed_manager_test.dart

```diff
@@ -197,43 +200,15 @@ Future<void> main() async {
     });
   });
 
-  group('Excluded sources', () {
-    setUp(() async {
-      final stackId = StackId();
-      final doc1 = Document(
-        documentId: DocumentId(),
-        stackId: stackId,
-        batchIndex: 1,
-        resource:
-            mockNewsResource.copyWith(sourceDomain: Source('www.nytimes.com')),
-      );
-      final doc2 = Document(
-        documentId: DocumentId(),
-        stackId: stackId,
-        batchIndex: 2,
-        resource:
-            mockNewsResource.copyWith(sourceDomain: Source('www.bbc.com')),
-      );
-      await docRepo.updateMany([doc1, doc2]);
-      await sourcePreferenceRepo.save(
-        SourcePreference(Source('www.bbc.com'), PreferenceMode.excluded),
-      );
-    });
-
+  group('Source preferences', () {
     tearDown(() async {
-      await docBox.clear();
+      await sourceBox.clear();
+      engine.resetCallCounter();
     });
 
     test('addExcludedSource', () async {
-      final excludedSources = {
-        Source('www.bbc.com'),
-        Source('www.nytimes.com')
-      };
-      final source1 = Source('www.bbc.com');
-      final source2 = Source('www.nytimes.com');
-
-      final response1 = await mgr.addExcludedSource(source1);
-      final response2 = await mgr.addExcludedSource(source2);
+      final response1 = await mgr.addExcludedSource(Source('bbc.com'));
+      final response2 = await mgr.addExcludedSource(Source('nytimes.com'));
```

Comment:
:wrench: since i just worked on this topic in #314, we need to replace the domains:
```suggestion
      final response1 = await mgr.addExcludedSource(Source('example.com'));
      final response2 = await mgr.addExcludedSource(Source('example.org'));
```
similarly for the other occurences below.